src/components/TreeNode/index.js

```diff
@@ -104,11 +92,9 @@ class TreeNode extends PureComponent {
         const {node, style} = this.props;
         const decorators = this.decorators();
         const animations = this.animations();
-
         return (
-            <Li ref={this.topLevelRef}
-                style={style.base}>
-                <NodeHeader {...{decorators, animations, node, style}} onClick={() => this.onClick()}/>
+            <Li style={style.base}>
+                <NodeHeader id="button" {...{decorators, animations, node, style}} onClick={() => this.onClick()}/>
```

Comment:
the `id` should be unique to avoid issues with the HTML, the user of this package can get issues with this ID.